designsafe/apps/api/views.py

```diff
@@ -155,3 +157,39 @@ def post(self, request):
             },
         )
         return HttpResponse("OK", status=202)
+    
+
+
+class SystemQueueProxyApi(BaseApiView):
+    """"
+    Proxy API for fetching system queue data from TAP. 
+    """
+
+    def get(self, request, hostname):
+        url = f"{settings.TAP_API_STATUS}/{hostname}"
+        
+        try:
+            response = requests.get(url, timeout=10)
+            response.raise_for_status()
+            return JsonResponse(response.json(), safe=False)
```

Comment:
Let's not use `safe=False`, but rather return a dict for `JsonResponse` with `{'response': ..., 'status': ...}`